src/coreclr/tools/superpmi/superpmi-shared/methodcontext.cpp

```diff
@@ -4735,6 +4735,86 @@ CORINFO_FIELD_HANDLE MethodContext::repGetFieldInClass(CORINFO_CLASS_HANDLE clsH
     return result;
 }
 
+void MethodContext::recGetTypeLayout(GetTypeLayoutResult result, CORINFO_CLASS_HANDLE typeHnd, CORINFO_TYPE_LAYOUT_NODE* nodes, size_t numNodes)
```

Comment:
Is it confusing that this takes `size_t numNodes` instead of `size_t* numNodes` like the API itself? (although presumably it shouldn't alter the number)

Reply:
Hmm, I initially had it as the pointer but realized this shouldn't be modified by the recording part, and changed it. I think I'll leave it like that.